becquerel/core/calibration.py

```diff
@@ -637,15 +748,18 @@ def __eq__(self, other):
             )
         if len(self.params) != len(other.params):
             return False
-        return (self.expression == other.expression) and np.allclose(
-            self.params, other.params
+        return (
+            (self.expression == other.expression)
+            and np.allclose(self.params, other.params)
+            and np.allclose(self.aux_params, other.aux_params)
         )
 
     def copy(self):
         """Make a complete copy of the calibration."""
         cal = Calibration(
             self.expression,
             self.params,
+            aux_params=self.aux_params,
             inv_expression=self.inv_expression,
             domain=self.domain,
             rng=self.range,
```

Comment:
Do any of these args need `.copy()`?

Reply:
I don't think so -- `params` and `aux_params` are copied during `__init__` (it's hidden in the setters like [this one](https://github.com/lbl-anp/becquerel/blob/new-calibration/becquerel/core/calibration.py#L635)). For `attrs` I perform a deep copy [here in its setter](https://github.com/lbl-anp/becquerel/blob/new-calibration/becquerel/core/calibration.py#L675). The others are strings or tuples.